src/clj_gcp/storage/core.clj

```diff
@@ -38,7 +39,16 @@
 ;;,---------------
 ;;| GCSStorageClient
 ;;`---------------
+(defn gcs-exists
```

Comment:
Feels a bit weird since it's not really checking it exists... but maybe that's OK due to nil-punning.
It's a shame the other function is called `get-blob` when in fact it's doing a bit more than simply getting the blob where as that is exactly what this function is doing